operators/cp4ba-operator/base/catalogsource.yaml

```diff
@@ -0,0 +1,106 @@
+################################################################################
+#
+# Licensed Materials - Property of IBM
+#
+# (C) Copyright IBM Corp. 2019. All Rights Reserved.
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+#
+###############################################################################
+# CP4BA 22.0.1 catalog
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: ibm-cp4a-operator-catalog
+  namespace: openshift-marketplace
+spec:
+  displayName: ibm-cp4a-operator
+  publisher: IBM
+  sourceType: grpc
+  image: icr.io/cpopen/ibm-cp-automation-catalog@sha256:f0b137d20ed52cc5bec8dd7ca03fe57e824cf30418803d421c6b50f4d7f5297e
```

Comment:
This comment will be applicable to the remaining spec.image entries in this file.

Since Customisation on overlays/latest already performs the required substitution - you can replace the values here with something like replaceme to make it obvious to the user this is a configurable parameter. (I've left a similar comment in the subscription.yaml file)